types/src/front/lib/actions/registry.ts

```diff
@@ -159,7 +159,7 @@ export const DustProdActionRegistry = createActionRegistry({
   "assistant-v2-websearch": {
     app: {
       workspaceId: PRODUCTION_DUST_APPS_WORKSPACE_ID,
-      appId: "78bda07b39",
+      appId: "098b515f8e",
```

Comment:
?

Reply:
correcting a mistake from a previous PR (at a time at which the run implementation was not done so app was not called)